homeassistant/components/lcn/__init__.py

```diff
@@ -25,11 +25,15 @@
 CONF_TRANSITION = 'transition'
 CONF_DIMMABLE = 'dimmable'
 CONF_CONNECTIONS = 'connections'
+CONF_MOTOR = 'motor'
 
 DIM_MODES = ['STEPS50', 'STEPS200']
 OUTPUT_PORTS = ['OUTPUT1', 'OUTPUT2', 'OUTPUT3', 'OUTPUT4']
 RELAY_PORTS = ['RELAY1', 'RELAY2', 'RELAY3', 'RELAY4',
-               'RELAY5', 'RELAY6', 'RELAY7', 'RELAY8']
+               'RELAY5', 'RELAY6', 'RELAY7', 'RELAY8',
+               'MOTORONOFF1', 'MOTORUPDOWN1', 'MOTORONOFF2', 'MOTORUPDOWN2',
```

Comment:
Why do we extend the relay ports? Will the cover be exposed as a light or switch too?

Reply:
No. The relais ports of the LCN hardware modules are also used for motor controls (with a certain wiring described in the docs). I added the constants just for completeness. They are just synonyms for the relais ports.